arangod/RocksDBEngine/RocksDBOptionsProvider.h

```diff
@@ -44,6 +42,10 @@ struct RocksDBOptionsProvider {
   rocksdb::BlockBasedTableOptions const& getTableOptions() const;
   virtual rocksdb::ColumnFamilyOptions getColumnFamilyOptions(
       RocksDBColumnFamilyManager::Family family) const;
+  void resetVPackComparator(
+      std::unique_ptr<rocksdb::Comparator> newComparator) {
+    _vpackCmp.reset(newComparator.release());
```

Comment:
nit: I find this variant slightly more intuitive:
```suggestion
    _vpackCmp = std::move(newComparator);
```